src/test/java/org/datadog/jenkins/plugins/publishers/DatadogComputerPublisherTest.java

```diff
@@ -0,0 +1,44 @@
+package org.datadog.jenkins.plugins.publishers;
```

Comment:
```suggestion
package org.datadog.jenkins.plugins.datadog.publishers;
```
Your test is in the wrong package

Reply:
Ah, I see